tests/disaster_recovery_3.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+import os
+import shutil
+import signal
+import time
+from TestHarness import Cluster, TestHelper, Utils, WalletMgr
+from TestHarness.Node import BlockType
+
+###############################################################
+# disaster_recovery - Scenario 3
+#
+# Create integration test with 4 nodes (A, B, C, and D) which each have their own producer and finalizer. The finalizer
+# policy consists of the four finalizers with a threshold of 3. The proposer policy involves all four proposers.
+#
+# - At least two of the four nodes should have a LIB N and a finalizer safety information file that locks on a block
```

Comment:
Probably you should say something like "make a condition such that at least two of the four nodes should have ..."